### DISCUSSION TITLE
Any advance/fix with Personal Results?

### DISCUSSION THREAD

**Discussion by Goldensit0** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#878] [Q&A]
I had the same issue as some other people, I believe, regarding the Personal Results that are no longer enabled, or at least I wasn't able to turn them on. My phone recieves a notification but it crashes and I have no idea why.

From what I found, I think I understood that this is no longer a thing? It is no longer possible to have personal results on GAUDC?

I have tried all guides I found, but it seems to be useless.

**Comment by sumeshir26** [comment]
You can try this: https://github.com/Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client/issues/784#issuecomment-1465096952

**Reply by Goldensit0** [reply]
Thanks, that worked for the notification crash but not for the personal results. However, I did the process mentioned below and it worked just fine, thanks!

**Reply by sumeshir26** [reply]
@Goldensit0 Glad to hear it worked for you!
